### DISCUSSION TITLE
vault & conjur

### DISCUSSION THREAD

**Discussion by pcf-user** [concourse/concourse#7192] [Help & Support]
Hi Team,

Concourse Version: 7.x.x

Currently we are using vault for managing our secrets and planning to start using Conjur. Is it possible to have both credential managers at the same time?

**Comment by taylorsilva** [comment]
Currently only one creds manager is configurable. The `var_sources` RFC is meant to address this: https://github.com/concourse/rfcs/blob/master/039-var-sources/proposal.md
It allows you to configure different and multiple creds managers per pipeline.
